bin/index.js

```diff
@@ -42,7 +43,7 @@ function parseOptions() {
     flow: false,
     esbuild: false,
     babel: false,
-    coverage: false,
+    coverage: isTruthy(process.env.EXODUS_TEST_COVERAGE),
```

Comment:
Can just use `![undefined, '', '0'].includes(process.env.EXODUS_TEST_COVERAGE),` as below and simplify this

Reply:
Or deduplicate that in-file
Will do that shortly

Reply:
Less expressive and passes on 'off'/'false'. Sometimes it's nice to keep the env var in code even if it's switched off for documentation purposes

Reply:
happy to reverse the check to an `isFalsy` though and move it into the index file

Reply:
https://github.com/ExodusMovement/test/pull/10/commits/c631da931e856120e244467ff41128549e965cb3

Reply:
I added `getEnvFlag` to main